src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -195,6 +195,12 @@ LedgerEntryIsValid::checkIsValid(TrustLineEntry const& tl,
     {
         return "TrustLine asset is invalid";
     }
+    if (tl.asset.type() == ASSET_TYPE_POOL_SHARE && tl.ext.v() == 1 &&
+        (tl.ext.v1().liabilities.buying != 0 ||
+         tl.ext.v1().liabilities.selling != 0))
+    {
+        return "TrustLine has pool share liabilities";
```

Comment:
This error message is weird. How about "Pool share TrustLine has liabilities"?

Reply:
Fixed